docs/debug.md

```diff
@@ -33,7 +33,7 @@ You can customize the core dump pattern with the following options
 
 There is a debug snap that can be installed with:
 
-    $ sudo snappy install snappy-debug
+    $ sudo snap install snappy-debug
```

Comment:
But it'll be back eventually, right? We'll have to add all this back in at that point.